public/app/AppWrapper.tsx

```diff
@@ -107,18 +108,20 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
                 >
                   <ModalsProvider>
                     <GlobalStyles />
-                    {config.featureToggles.commandPalette && <CommandPalette />}
+                    {!isDashboardPubliclyViewed() && config.featureToggles.commandPalette && <CommandPalette />}
                     <div className="grafana-app">
                       <Router history={locationService.getHistory()}>
-                        {ready && <>{newNavigationEnabled ? <NavBarNext /> : <NavBar />}</>}
+                        {!isDashboardPubliclyViewed() && ready && (
+                          <>{newNavigationEnabled ? <NavBarNext /> : <NavBar />}</>
+                        )}
                         <main className="main-view">
                           {pageBanners.map((Banner, index) => (
                             <Banner key={index.toString()} />
                           ))}
 
                           <AngularRoot />
-                          <AppNotificationList />
-                          <SearchWrapper />
+                          {!isDashboardPubliclyViewed() && <AppNotificationList />}
```

Comment:
This didn't _seem_ needed for a public dashboard page. Wasn't sure though.

Reply:
We can ditch the search, but we still probably want AppNotificationList so we can alert the user on error